### ISSUE TITLE
Output cell is too small for the DataFrame and shows scrollbar

### ISSUE THREAD

**Issue by jonasjonker** [fonsp/Pluto.jl#996]
When I output a dataframe, the output cell is to small for the DataFrame and therefor has a scrollbar.  I tried it in two browsers (Firefox and Brave). I assume this is not a feature. What is the best way for me to help find the cause? 

![2021-03-14-124518_728x372_scrot](https://user-images.githubusercontent.com/24474645/111086984-7a768480-851f-11eb-9513-e106a13d420e.png)

**Comment by fonsp** [comment]
This is a feature :)

**Comment by jonasjonker** [comment]
Oke :)

**Comment by GiggleLiu** [comment]
![image](https://user-images.githubusercontent.com/6257240/123518648-d9967a00-d674-11eb-82bf-d769d9521165.png)

Hi, is there an easy solution to remove the scrollbar here?

```julia
md"""
When $m<b$, the mapping is $(m, b) \rightarrow (b, b)$. Let `m` and `b` be 64 bit integers.
If all states have equal probability, their information entropies are $128\log 2$ and $64 \log 2$
"""
```

Pluto@v0.14.7

**Comment by holomorphism** [comment]
@GiggleLiu
I was also wondering about the same issue.

When I checked the MathJax formula, the height of the element "mjx-assistive-mml" was exceeding the element "pluto-output", so I think this might be the cause.

![image](https://user-images.githubusercontent.com/3076585/123520050-afcb6a80-d6e9-11eb-98ee-69692aa0f90d.png)

I wasn't sure if this was a Pluto issue or a MathJax issue, so I hadn't reported this issue yet.


After adding the following custom css, the extra scrollbars are no longer displayed, so I have adopted this workaround when publishing notebooks as html.

```julia
"""
<style>
mjx-assistive-mml { display: none !important; }
</style>
""" |> HTML
```

However, I am not sure if hiding "mjx-assistive-mml" will cause any problems.
